packages/@angular/cli/lib/cli/index.ts

```diff
@@ -1,3 +1,9 @@
+// We don't actually require AJV, but there is a bug with NPM and peer dependencies that is
+// whose workaround is to depend on AJV.
+// See https://github.com/angular/angular-cli/issues/9691#issuecomment-367322703 for details.
+// We need to add a require here to satisfy the dependency checker.
+// require('ajv');
```

Comment:
Could you add a TODO here to make sure we get it in our reminders?

Reply:
Done